packages/components/esbuild.config.js

```diff
@@ -1,7 +1,74 @@
 // @ts-nocheck
 /* eslint-disable import/no-extraneous-dependencies */
-const postCssPlugin = require("@deanc/esbuild-plugin-postcss")
-const esbuild = require("esbuild")
+import { existsSync, mkdirSync, readdirSync, statSync, writeFileSync } from "fs"
+import { join } from "path"
+import postCssPlugin from "@deanc/esbuild-plugin-postcss"
+import autoprefixer from "autoprefixer"
+import esbuild from "esbuild"
+import tailwindcss from "tailwindcss"
+import glob from "tiny-glob"
+
+const dist = join(process.cwd(), "dist")
+
+if (!existsSync(dist)) {
+  mkdirSync(dist)
+}
+
+// const entryPoints = readdirSync(join(process.cwd(), "src"))
+//   .filter(
+//     file =>
+//       file.endsWith(".ts") &&
+//       statSync(join(process.cwd(), "src", file)).isFile()
+//   )
+//   .map(file => `src/${file}`)
+
+// esm output bundles with code splitting
```

Comment:
What's happening here?

Reply:
Ah leftover code from an example, I resolved this with the glob